nimblepayment.php

```diff
@@ -180,7 +180,7 @@ public function renderForm()
 				),
 				array(
 						'id_option' => 'sandbox',
-						'name' => 'Sandbox'
+						'name' => 'Pruebas'
```

Comment:
Hi @Quetzacoalt91 

Ok! We have added the word in english with the translation system, and we have updated the translation file for spanish. Is it correct?

Thanks!